docs/UserGuide.md

```diff
@@ -412,7 +416,8 @@ _Details coming soon ..._
 5. **Phone Number Validation Lacks Support for International Formats**: The current phone number validation requires that phone numbers only contain numeric characters and be at least 3 digits long. However, this limitation may not adequately support users with international contacts, as it lacks flexibility for common international number formats that include country codes (e.g., +65 91234567 or +1 123-456-7890). This may pose a problem for students who frequently communicate with international friends. Future improvements aim to support international phone number formats by allowing optional country codes and special characters such as + and -. In the meantime, users are advised to store international numbers in a consistent, numeric-only format (e.g., without + or spaces) to ensure compatibility with the current app constraints.
-6. **Lack of Support for Multiple Phone Numbers per Contact**: Currently, each contact can only store a single phone number. This limitation may be inconvenient for users who need to save multiple numbers for a single person, such as a mobile and home number or alternative international contact numbers. Future enhancements may include support for storing multiple phone numbers for each contact. Until then, users who need to save additional phone numbers for a contact may consider creating a separate entry with a label in the contact’s name (e.g., "Alex Yeoh - Home" and "Alex Yeoh - Mobile") 
+6. **Lack of Support for Multiple Phone Numbers per Contact**: Currently, each contact can only store a single phone number. This limitation may be inconvenient for users who need to save multiple numbers for a single person, such as a mobile and home number or alternative international contact numbers. Future enhancements may include support for storing multiple phone numbers for each contact. Until then, users who need to save additional phone numbers for a contact may consider creating a separate entry with a label in the contact’s name (e.g., "Alex Yeoh - Home" and "Alex Yeoh - Mobile")
+7. **Find Command Does Not Persist Filtered State for Persons**: Currently, executing commands such as `add` or `edit` resets the person filter, causing the UI to refresh. This can be inconvenient for users wishing to add new entries to an already filtered list of persons. Future enhancements aim to retain the person filter state for such commands.
```

Comment:
```suggestion
7. **Find Command Does Not Persist Filtered State for Persons**: Currently, executing commands such as `add` or `edit` resets the person filter, causing the UI to refresh. This can be inconvenient for users wishing to execute commands to the existing filtered list of persons. Future enhancements aim to retain the person filter state for such commands.
```